### DISCUSSION TITLE
How to keep duplicate theme in sync with master ?

### DISCUSSION THREAD

**Discussion by Twilight-Computer** [givanz/Vvveb#110] [General]
I have created a theme (using duplicate) and now finding it is out of date to the master "landing" theme.
Should I have created a "child" theme and / or what are the suggestions and recommendations to keep local themes up to date ?

**Comment by givanz** [comment]
The simple and easy method is to manually merge the changes from the customized files using a tool such as https://meldmerge.org/ and overwrite all other files.

```bash
meld public/themes/landing public/themes/duplicate-theme
```

Child themes are themes that override or change parts from a parent theme using `data-v-copy-from` attribute http://dev.vvveb.com/theme-introduction#child-themes

Parts that are copied from parent theme will still overwrite the content saved in the child and the update issue will not be solved.

Implementing a child theme also takes time as you need to create a new template (html file) for each page you use from the parent theme with a data-v-copy element along with the custom child elements.
